internal/service/notification.go

```diff
@@ -111,40 +113,56 @@ func EncryptAES(key []byte, plaintext string) string {
 */
 
 // send push notification to device via pushy.me service
-func SendPushyMe(msg Notification, device model.Device) (err error) {
-	if len(device.TokenPush) < 1 {
-		return errors.New("wrong push token for pushy.me")
-	}
-	// https://stackoverflow.com/questions/40123319/easy-way-to-encrypt-decrypt-string-in-android
-	// use first 32 characters from device token as basic encryption measure in transit
-	//encryptKey := []byte(device.Token[0:32])
-	//msg.Title = EncryptAES(encryptKey, msg.Title)
-
-	log.Printf("Sending pushy.me msg to %v", device.Name)
-	pushReqRaw := PushyMeReq{
-		To:           device.TokenPush,
-		Notification: msg,
-	}
-	pushReq, err := json.Marshal(&pushReqRaw)
-	if err != nil {
-		log.Printf("failed preparing msg for push notification: %v", err)
-		return err
+//func SendPushyMe(msg Notification, device model.Device) (err error) {
+//	if len(device.TokenPush) < 1 {
+//		return errors.New("wrong push token for pushy.me")
+//	}
+//	// https://stackoverflow.com/questions/40123319/easy-way-to-encrypt-decrypt-string-in-android
+//	// use first 32 characters from device token as basic encryption measure in transit
+//	//encryptKey := []byte(device.Token[0:32])
+//	//msg.Title = EncryptAES(encryptKey, msg.Title)
+//
+//	log.Printf("Sending pushy.me msg to %v", device.Name)
+//	pushReqRaw := PushyMeReq{
+//		To:           device.TokenPush,
+//		Notification: msg,
+//	}
+//	pushReq, err := json.Marshal(&pushReqRaw)
+//	if err != nil {
+//		log.Printf("failed preparing msg for push notification: %v", err)
+//		return err
+//	}
+//
+//	c := &http.Client{}
+//	reqUrl := "https://api.pushy.me/push?api_key=" + config.PUSHY_ME_SECRET
+//	r, err := http.NewRequest("POST", reqUrl, bytes.NewBuffer(pushReq))
+//	if err != nil {
+//		log.Printf("fail when creating request for api.pushy.me: %v", err)
+//		return err
+//	}
+//	r.Header.Set("Content-Type", "application/json")
+//	res, err := c.Do(r)
+//	if err != nil {
+//		log.Printf("fail when sending request to api.pushy.me: %v", err)
+//		return err
+//	}
+//	defer res.Body.Close()
+//	log.Printf("HTTP %v @ api.pushy.me", res.StatusCode)
+//	return nil
+//}
+
+func PushoverNotification(msg Notification, device model.Device) error {
+	app := pushover.New(config.PUSHOVER_API_TOKEN)
+	recipient := pushover.NewRecipient(config.PUSHOVER_USER_KEY)
+	message := &pushover.Message{
+		Message:    msg.Msg,
+		Title:      msg.Title,
+		Priority:   pushover.PriorityNormal,
+		DeviceName: device.Name,
```

Comment:
You don't need device name property if notification must go to all pushover devices